### ISSUE TITLE
Feature request: JSON decode

### ISSUE THREAD

**Issue by Finesse** [particle-php/Filter#56]
It would be great if Filter supports this:

```php
$filter = new Filter();
$filter->value('jsonData')->decodeJSON(true /* json_decode arguments here */);
$result = $filter->filter([
    'jsonData' => '{"foo": "bar"}'
]);
var_dump($result);
/*
[
    'jsonData' => ['foo' => 'bar']
]
*/
```

I can make a pull request if you like it.

**Comment by rick-nu** [comment]
@FinesseRus Nice, I like it! :+1: If you want to make a pull request than all the credits go to you! :wink:

Else I'll take a look at it, because I think it's nice to include in the library :+1:

**Comment by Finesse** [comment]
@RickvdStaaij I made a pull request for this issue: #58

**Comment by rick-nu** [comment]
Released in [v1.5.0](https://github.com/particle-php/Filter/releases/tag/v1.5.0), thank you! :)
